src/api/firebase.js

```diff
@@ -79,3 +85,10 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export async function checkListToken(listId) {
```

Comment:
@tinazarb @louieknolle Did you determine what was needed here via the Firebase docs?